test.js

```diff
@@ -1,51 +1,100 @@
 const net = require('net');
 const test = require('ava');
-const pify = require('pify');
 
 const { findFreePort, isPortFree } = require('.');
 
-test('should return the passed port value', async (t) => {
+const requestedPort = 3010;
+const requestedHost = '127.0.0.1';
+const defaultHost = '0.0.0.0';
+
+const wrapper = (t, run) => new Promise(resolve => run(t, resolve));
+
+test.serial('should return the passed port value', async (t) => {
```

Comment:
We have two funcs in the package, so it would be great to create to `describe`-blocks where one func's tests will be separated from another one.

Not sure that ava has this feature, but check it please.

Reply:
There is a current discussion about grouping tests [here](https://github.com/avajs/ava/issues/222), but it is not supported yet.

Reply:
Ah, okay.